addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -57,6 +57,9 @@ export class OutOfFocusService {
             title: notificationTitle,
             type: "info",
         });
+        if (thread?.model === "discuss.channel") {
```

Comment:
`thread` can be empty?

Reply:
Yes, the thread is optional. Actually this is always `message.thread`, so I made more diff to make 2nd parameter `thread` deprecated.